tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -82,9 +82,9 @@ def alter_gis_model(self, migration_class, model_name, field_name,
         operation = migration_class(*args)
         new_state = project_state.clone()
         operation.state_forwards('gis', new_state)
+        self.current_state = new_state
```

Comment:
Yes, these tests fail on spatialite with the following message:

    OperationalError: table "gis_neighborhood" already exists

Moving that line up fixes the problem. I remember it took me a while to even identify the problem when I was writing the lookups. I can try to look further into why this happens, but honestly I don't understand what is going on. Any hints/help regarding this are most welcome.